packages/interval/index.d.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="react" />
```

Comment:
this file needs to be added to `files` section of `package.json`, otherwise it will not be included into a package.

To check that locally you can run `yarn pack` (or `npm pack`) inside folder and then check contents of resulting `react-interval-*.tgz`

Reply:
Done